src/Adapter/MSTestAdapter.PlatformServices/Services/SettingsProvider.cs

```diff
@@ -40,6 +41,15 @@ public static MSTestAdapterSettings Settings
     public static void Reset() => s_settings = null;
 #endif
 
+    internal static void Load(IConfiguration configuration)
+    {
+#if !WINDOWS_UWP
+#pragma warning disable IDE0022 // Use expression body for method
```

Comment:
I think that this pragma can be removed or?

Reply:
when i remove it it request an arrow function and when i do it I get another error because of `#if`